<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShow</class>
 <widget class="QDialog" name="SlideShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1">
   <item>
    <widget class="QWidget" name="wgt_SlideShow" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PicStateBtn" name="pbtn_Play">
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PicButton" name="pbtn_Close">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="wgt_Pre" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PicButton" name="pbtn_Pre">
           <property name="text">
            <string>前一个</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PicAnimationWgt" name="wgt_PicAnimation" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="wgt_Next" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PicButton" name="pbtn_Next">
           <property name="text">
            <string>后一个</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_PreviewShow" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QWidget" name="wgt_PreviewParent" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <widget class="PreviewListWgt" name="lwgt_Preview">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>16777215</width>
           <height>110</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PicButton</class>
   <extends>QPushButton</extends>
   <header location="global">picbutton.h</header>
  </customwidget>
  <customwidget>
   <class>PicStateBtn</class>
   <extends>QPushButton</extends>
   <header location="global">picstatebtn.h</header>
  </customwidget>
  <customwidget>
   <class>PicAnimationWgt</class>
   <extends>QWidget</extends>
   <header>picanimationwgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewListWgt</class>
   <extends>QListWidget</extends>
   <header location="global">previewlistwgt.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
